pkgs/development/python-modules/starlette/default.nix

```diff
@@ -46,12 +46,15 @@ buildPythonPackage rec {
   checkInputs = [
     aiosqlite
     databases
+    pytest-asyncio
+    pytest-cov
```

Comment:
Coverage is another thing we don't really need to test in nixpkgs.

Reply:
Patch out these lines in postPatch instead: https://github.com/encode/starlette/blob/ea1990415ebef516e17f2cb724a1c4384c479597/setup.cfg#L20-L22

Reply:
What's so bad about having `pytest-cov` in `checkInputs` if it saves us from patching?

Reply:
It's just not helpful and adds noise to the build process and the patching would be quite stable in most cases.

```
sed -i "s/--cov/d" setup.cfg
```

Reply:
Added, thanks.

Reply:
In fact that will be needed for 0.14.2 but not for 0.13.8.